pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -3,7 +3,7 @@
 let
   inherit (args) stdenv makeWrapper;
   inherit (stdenv) lib isDarwin;
-  inherit (lib) overrideDerivation;
+  inherit (lib) overrideDerivation compose;
```

Comment:
where is this method used here?

Reply:
Hmmm... you're right, doesn't looks like that expression was actually in use there. I'll drop the inherit.